integrations/jira/jira/client.py

```diff
@@ -54,11 +52,103 @@ def __init__(self, jira_url: str, jira_email: str, jira_token: str) -> None:
             self.jira_api_auth = BasicAuth(self.jira_email, self.jira_token)
 
         self.api_url = f"{self.jira_rest_url}/api/3"
+        self.teams_base_url = f"{self.jira_url}/gateway/api/public/teams/v1/org"
```

Comment:
whats the `/gateway` prefix? asking because when this integration installed using OAuth2, the jira url will be in the form of https://api.atlassian.com/ex/jira/<cloud_id> so pls make sure that valid

Reply:
https://sitename.atlassian.net is the documented prefix however, this works with https://api.atlassian.net as well